javascript/javascript3/week1/Weather App/main.js

```diff
@@ -0,0 +1,77 @@
+let inputValue = document.querySelector(".inputValue");
+let buttonForInput = document.querySelector(".buttonForInput");
+let buttonLocation = document.querySelector(".buttonLocation");
+
+let city = document.querySelector(".city");
+let temp = document.querySelector(".temp");
+
+let iFTWeather = document.querySelector(".iFTWeather");
+let iconimg = document.createElement("img");
+let windspeed = document.querySelector(".windspeed");
+let hClowdy = document.querySelector(".hClowdy");
+let SunrisTo = document.querySelector(".SunrisTo");
+let SunsetTo = document.querySelector(".SunsetTo");
+
+buttonForInput.addEventListener("click", () => {
+  const apiUri = `http://api.openweathermap.org/data/2.5/weather?q=${inputValue.value}
+&appid=3f9781b65d09b084b2f1f07fc5f75557`;
+
+  async function weatherApp() {
+    const response = await fetch(apiUri);
+    const date = await response.json();
+    console.log(date);
+
+    let cityName = ` ${date["name"]}, ${date["sys"]["country"]}`;
+    let temp1 = `Temperature: ${
+      date["main"]["temp"]}`;
```

Comment:
Not sure why there is a line-break here? 🤔

Reply:
By mistake sorry, I fix it